app/services/organizations/importers/google_csv_import_service.rb

```diff
@@ -3,41 +3,61 @@
 module Organizations
   module Importers
     class GoogleCsvImportService
+      Status = Data.define(:success?, :count, :no_match, :errors)
```

Comment:
Just a nit: I think a new line could separate the method from the Status definition to make it easier to read.

Reply:
New line is removed with standard:fix. But I do agree

Reply:
oh weird! Ok well looks like we have to keep it as is otherwise linting will fail